.github/workflows/ios.yml

```diff
@@ -55,4 +55,4 @@ jobs:
 
       - name: Build SDK
         run: |
-          build_scripts/ios/build.sh -b ios_build -s .
+          python scripts/gha/build_ios_tvos.py -b ios_tvos_build -s .
```

Comment:
Also, update the file name & workflow name.

Reply:
Will take care of this in a new PR since I will be removing the workflow files from this one.